docs/docsource/operation/XDM_GUI/location_management/adding_a_transient_source_storage_location.rst

```diff
@@ -0,0 +1,60 @@
+.. _Adding a Transient Source Storage Location:
```

Comment:
this file and the next file do not use `|product|` for zenko references, not sure if intentional.

Reply:
This file is basically a place-holder so that the cross-references hold up. I will revisit this file in future tickets (either to edit it or delete it).